doc/man3/X509_get_subject_name.pod

```diff
@@ -68,7 +68,9 @@ X509_get_subject_name(), X509_get_issuer_name(), X509_REQ_get_subject_name()
 and X509_CRL_get_issuer() return an B<X509_NAME> pointer.
 
 X509_subject_name_hash() and X509_issuer_name_hash()
-return the respective hash value or 0 on failure.
+return the first four bytes of the SHA1 hash value,
```

Comment:
Please add X509_NAME_hash_ex and X509_NAME_hash here as well.

Reply:
Oops, done.
Also squashed the recent fixup commits.